Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
```

Comment:
Actually I take that back -we no longer want the part of this that's cloning gi-bill-comparison-tool repo, and that is what is causing jenkins to fail - because its winding up running rubocop on that subdirectory.